ha/core/controllers/pcs/node_controller.py

```diff
@@ -71,7 +71,28 @@ def stop(self, node_id: str, timeout: int, **op_kwargs) -> dict:
             ([dict]): Return dictionary. {"status": "", "output": "", "error": ""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        check_cluster = op_kwargs.get("check_cluster") if op_kwargs.get("check_cluster") is not None else True
+        # Get the node_name (pvtfqdn) fron nodeid and raise exception if node_id is not valid
+        node_name = self._get_node_name(node_id=node_id)
```

Comment:
Please check PR https://github.com/Seagate/cortx-ha/pull/547/files.
get_node_name() has been added to config_manager.py file, we should use the same.

Reply:
get_node_name() moved to config_manager.py file